src/pages/Main/Main.js

```diff
@@ -1,13 +1,69 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { withRouter } from 'react-router-dom';
 import styled from 'styled-components';
-
+import axios from 'axios';
+import BtnImage from './Btn';
 function Main() {
+  const [User, setUser] = useState({
```

Comment:
- 컴포넌트 관련한 이름만 첫글자를 대문자로 시작합니다! 이건 state 이름이므로, 소문자로 작성해주세요!
```suggestion
  const [user, setUser] = useState({
```
- 그리고 이 state 값을 굳이 객체로 한번 감싸는 이유가 있으신가요? 없다면 굳이 객체로 한번 바깥에서 감싸지 마세요~ 오히려 업데이트할 때 과정이 번거로워 집니다!
```jsx
const [productList, setProductList] = useState([])
```